apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-example.component.html

```diff
@@ -0,0 +1,161 @@
+<h3>Date Picker with Reactive Form.</h3>
+<fdp-form-group #ffg [formGroup]="datePickerForm" [object]="data" (onSubmit)="save(ffg)" [multiLayout]="true">
+    <fdp-form-field
+        #ffl1
+        [id]="'birthday'"
+        zone="zLeft"
+        rank="1"
+        [required]="true"
+        [placeholder]="'Enter your birthday'"
+        [label]="'Birth Date:'"
```

Comment:
attributes with strings or booleans do not need to be angular attributes

Reply:
Done